docs/private-dns/server-and-settings/access.md

```diff
@@ -9,20 +9,24 @@ Blocked requests will not be displayed in the Query Log and are not counted in t
 
 ## How to set it up
 
-- **Allowed clients.** This allows you to specify which clients can use your DNS server.
+### Allowed clients
 
-This setting has the highest priority. For example, if the same IP address is on both the denied and allowed list, it will still be allowed.
+This setting  allows you to specify which clients can use your DNS server. It has the highest priority. For example, if the same IP address is on both the denied and allowed list, it will still be allowed.
```

Comment:
```suggestion
This setting allows you to specify which clients can use your DNS server. It has the highest priority. For example, if the same IP address is on both the denied and allowed list, it will still be allowed.
```